hokusai/development.yml

```diff
@@ -10,7 +10,7 @@ services:
     depends_on:
       - kaws-mongodb
   kaws-mongodb:
-    image: mongo:4.0
+    image: mongo:4.2
     ports:
       - "27017:27017"
-    command: ["--storageEngine=mmapv1", "--quiet", "--nojournal"]
```

Comment:
MMAPv1 has been removed as of `mongodb@4.2.`. Instead we need to use the default storage engine, `WiredTiger`, see mongo docs [here](https://docs.mongodb.com/v4.2/core/storage-engines/).